src/index.js

```diff
@@ -75,18 +65,20 @@ const FOOTER_DATA = [
         addNumberElement.innerText = "Add Number"
 
         // MAIN
-        const mainElement = document.createElement("main")
-        const buttonWrapperElement = document.createElement("div")
-        buttonWrapperElement.id = "button-wrapper"
+        const mainElement = document.querySelector("main")
+
+        BUTTONS_ARRAY.forEach((button, index) => {
+            const dialButtonElement = document.createElement("div")
+            dialButtonElement.className = "dial-button"
 
-        headerElement.appendChild(mainElement)
-        mainElement.appendChild(buttonWrapperElement)
+            const buttonValueElement = document.createElement('h2')
+            buttonValueElement.innerHTML = button.value
+            dialButtonElement.appendChild(buttonValueElement)
 
-        // FOOTER
-        const footerElement = document.createElement("footer")
-        const footerWrapperElement = document.createElement("div")
-        footerWrapperElement.id = "footer-wrapper"
+            const buttonCharactersElement = document.createElement('p')
```

Comment:
And then here, when `button.characters` is not undefined you could use `if` to perm this action. There is no need to create empty p element